scripts/routing.js

```diff
@@ -373,22 +394,41 @@ async function searchPlace() {
 				],
 				boundary_country: ["PT"],
 			});
+
 			// set the inner HTML after the animation has started
 			menu.innerHTML = `
                 <ul id="placeList">
                     <!-- Populate with the list here -->
                 </ul>
             `.trim();
 
+			// if there are no results, put a message saying that
+			if (!response || (response && response.features.length === 0)) {
```

Comment:
se estás a ver antes se é falsy, para quê ver logo a seguir se é truthy?
```suggestion
			if (!response || response.features.length === 0) {
```

Reply:
tinha escrito a condição da direita primeiro xd